### PR TITLE
[Ready] Nerfs(Not really) Engi printing and organized many designs

### PR DESCRIPTION
**Pull request by Trilbyspaceclone**
[Changelogs]
Nerfed Engi printer aka did nothing do to them being able to print off a unrestricted printer anyways

:cl: optional name here
del: Many engi flags on non-engi things
code: changed some code to be organized at a glance 
/:cl:

[why]
WEW that was messy!
Engi printer is powerful to be able to print legit all of medical/sci/sec gear < - Still is. But meh People hate their power being cut off

### CODE REVIEW

File: code/modules/research/designs/comp_board_designs.dm [hunk 1]
```diff
@@ -2,7 +2,7 @@
 
 /datum/design/board
 	name = "Computer Design ( NULL ENTRY )"
-	desc = "I promise this doesn't give you syndicate goodies!"
+	desc = "A blank compurter board!"
```
Comment (Poojawa) [inline code/modules/research/designs/comp_board_designs.dm]: computer

File: code/modules/research/designs/comp_board_designs.dm [hunk 2]
```diff
@@ -22,6 +22,26 @@
...
+	category = list("Computer Boards")
+	departmental_flags = DEPARTMENTAL_FLAG_ALL
+
+/datum/design/board/libraryconsole
+	name = "Computer Design (Library Console)"
+	desc = "Allows for the construction of circuit boards used to build a new library console."
+	id = "libraryconsole"
+	build_path = /obj/item/circuitboard/computer/libraryconsole
+	category = list("Computer Boards")
+	departmental_flags = DEPARTMENTAL_FLAG_ALL
```
Comment (Poojawa) [inline code/modules/research/designs/comp_board_designs.dm]: Why would service, security, medical, and science need to be able to print library book console circuits?

**Review by Poojawa — CHANGES_REQUESTED** [review]
I don't at all agree with depriving Engineering of it's core responsibility. If you want to nerf this, make it access locked

File: code/modules/research/designs/comp_board_designs.dm (hunk 2 above)
Comment (Trilbyspaceclone) [reply code/modules/research/designs/comp_board_designs.dm]: IDK it was all ALL flag'ed

Comment (Trilbyspaceclone) [reply code/modules/research/designs/comp_board_designs.dm]: Also thats console not printer, its prob to add in books/catalogs without needing to leave you department

**Comment by silicons** [comment]
Engineering is supposed to be able to print all circuitboard types and it is staying that way.

**Review by silicons — APPROVED** [review]

**Pull request merged**